src/stactools/naip/stac.py

```diff
@@ -137,6 +142,7 @@ def create_item(
         fname_date = fname.split("_")[5]
         dt = dateutil.parser.isoparse(fname_date)
 
+    dt = dt + timedelta(hours=16)  # UTC is +4 ET, so is around 9-12 AM in CONUS
```

Comment:
Hmm, this feels a little strange. Do we have information about the expected time zone of the `Calendar_Date` attribute an/dor the filename datetime, and if so can we set that on the `dt` and then use libraries to get it to UTC?

Reply:
Calendar_Date is just an ISO 8601 date, so there's no tz attached. isoparse generates a datetime in for midnight on that date with the tz as UTC. I would worry about this more if it weren't only collected only once every few years.